package.json

```diff
@@ -33,6 +33,7 @@
     "@popperjs/core": "^2.11.8",
     "@swimlane/ngx-charts": "^20.4.1",
     "@swimlane/ngx-datatable": "20.1.0",
+    "@popperjs/core": "^2.11.8",
```

Comment:
Is this required here?

Reply:
I think it's duplicated. I will remove one. line 33 also the same one.